torch/csrc/jit/codegen/cuda/scheduler/pointwise_utils.cpp

```diff
@@ -12,7 +12,8 @@ bool DomainMap::areAllInputIdsMappedTo(TensorView* input_tv, TensorView* tv)
   // Get concrete IDs for input root or rfactor domain
   std::unordered_set<IterDomain*> in_concrete_ids;
   for (auto in_id : input_tv->getMaybeRFactorDomain()) {
-    auto concrete = ca_map_.getConcreteMappedID(in_id, IdMappingMode::EXACT);
+    auto concrete =
```

Comment:
Is this related to the permissive map change? Don't remember what this function is trying to do, but why using the permissive map instead of the exact map?

Reply:
Exact mapping is sufficient for the pointwise scheduler, but permissive mapping is required to allow the transpose scheduler to analyze fusions like
```
T0[I0, b] + T1[b, I1]
```
We are already using permissive map in the `eraseIfMapped` below, and it is wrong if we don't keep it consistent here.

However, this bug was not discovered until today, because my change makes the permissive map and exact map selecting a different concrete domain.

Reply:
I was wondering what potential side effects could be by changing it to use the permissive map, but if I understand the logic here correctly, we are only looking at root domains and broadcasts are immediately ignored anyway, so the only impact this change could have is the transpose case as you mentioned. Am I right?

Reply:
Yes

Reply:
Can you please leave a comment about this?